CMakeLists.txt

```diff
@@ -157,7 +157,7 @@ if(BUILD_TESTING AND ABSL_BUILD_TESTING)
     include(CMake/Googletest/DownloadGTest.cmake)
   endif()
 
-  if (NOT ABSL_FIND_GOOGLETEST)
+  if (ABSL_USE_EXTERNAL_GOOGLETEST AND NOT ABSL_FIND_GOOGLETEST)
```

Comment:
Maybe I'm misunderstanding this since we rarely use CMake ourselves, but could/should this logic be merged into the logic on lines 135-141?

Reply:
Now when you asked, it's easier to justify a larger change.

I'm pretty sure the aliases could be moved into `NOT ABSL_FIND_GOOGLETEST` section, but you might need to ask someone else to confirm, as the change becomes non-trivial.

To clarify:

* those aliases are not needed for `ABSL_USE_EXTERNAL_GOOGLETEST` case anymore, since google/googletest#3155 has been merged.
* there is no harm to add the same alias twice, as long as it refers to the same target.
* the actual problem is `NOT ABSL_USE_EXTERNAL_GOOGLETEST` case; in that case `GTest::gtest` is an actual target, so it couldn't be re-added as an alias.

Reply:
Thanks. Looks good, should be merged soon.